### DISCUSSION TITLE
EXAMPLE - LARGE MODEL

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#946] [Q&A]
Hi. Does anyone has large, explicit, working model in .rad files  ? I mean model which contain 500 000 to 1000 000 elements. I would like test two xeon CPU time. 

I found .rad files  https://altairuniversity.com/learning-library/taurus-crash-model-learn-more-about-it/   but openradios found some errors in this files  also I tried other .rad files and all didn't run

**Comment by OlivierWienholtz** [comment]
Hi 
You can take the Yaris & Camri Models at the OpeRadioss Examples.

https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/17629185/Example+LS-DYNA+Format+Models

Best Regards
Olivier

**Comment by CinekPoland** [comment]
Hi . Thank you I will try it. I downloaded crash impact  of Toyota from
Altair and files had some errors. Maybe files from link which you sent me
will be better. Thank you.


W dniu środa, 4 stycznia 2023 Olivier Wienholtz ***@***.***>
napisał(a):
